<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QAptOfflineFetch</class>
 <widget class="QDialog" name="QAptOfflineFetch">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fetch Packages or Upgrade</string>
  </property>
  <widget class="QLineEdit" name="profileFilePath">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>270</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browseFilePathButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startDownloadButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/go-down.png</normaloff>
     <normalon>icons/go-down.png</normalon>
     <disabledoff>icons/go-down.png</disabledoff>
     <disabledon>icons/go-down.png</disabledon>
     <activeoff>icons/go-down.png</activeoff>
     <activeon>icons/go-down.png</activeon>
     <selectedoff>icons/go-down.png</selectedoff>
     <selectedon>icons/go-down.png</selectedon>icons/go-down.png</iconset>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/dialog-cancel.png</normaloff>icons/dialog-cancel.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select the profile file</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="statusProgressBar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>410</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="rawLogHolder">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>410</width>
     <height>210</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
  </widget>
  <widget class="QLabel" name="progressStatusDescription">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>53</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ready</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
